### DISCUSSION TITLE
Contribution section in the docs

### DISCUSSION THREAD

**Discussion by zaaakher** [spacedriveapp/spacedrive#2422] [Feature Request]
I'm in love with Spacedive. Thank you guys for everything. ♥

I suggest adding a contribution section in the docs to guide us through how to contribute with code, ui, general design, translations, etc.

**Comment by HeavenVolkoff** [comment]
Hi, thank you for the interest in Spacedrive. We currently have contributing docs [here](https://github.com/spacedriveapp/spacedrive/blob/main/CONTRIBUTING.md) and [here](https://www.spacedrive.com/docs/developers/prerequisites/environment-setup). They are not perfect, but should get you up to a point to be able to implement a bugfix or some simple new feature, if you have any question or think something is missing from them feel free to open an issue.
